### DISCUSSION TITLE
Support for reference table

### DISCUSSION THREAD

**Discussion by tta-scalefree** [ScalefreeCOM/datavault4dbt#32] [Ideas]
Hi datavault4dbt team,

I'd like to make a feature request: could you please add support for reference tables?

Best regards,
Trung

**Comment by tkirschke** [comment]
> Hi datavault4dbt team,
> 
> I'd like to make a feature request: could you please add support for reference tables?
> 
> Best regards, Trung


Hi Trung!

Thanks for requesting reference tables! They are definitely on the list of what we want to support next. I will let you know, as soon as we have something up and running!

Kind regards, 
Tim

**Comment by tkirschke** [comment]
Reference Tables are now a part of the reference data strategy, released in v1.1.0 . 
Check out [this ](https://github.com/ScalefreeCOM/datavault4dbt/wiki/Reference-Data)wiki page for more details!
